src/adera/Machines/Rocket.cpp

```diff
@@ -80,53 +79,25 @@ SysMachineRocket::SysMachineRocket(ActiveScene &scene) :
 //{
 //}
 
-void SysMachineRocket::update_physics()
+void SysMachineRocket::update_physics(ActiveScene& rScene)
 {
-    auto view = m_scene.get_registry().view<MachineRocket>();
+    auto view = m_scene.get_registry().view<MachineRocket, ACompTransform>();
 
     for (ActiveEnt ent : view)
     {
         auto &machine = view.get<MachineRocket>(ent);
```

Comment:
out of curiousity, does EnTT not have a mechanism to do something like:

for(auto const& [things, from, the, view] : m_scene.get_registry().materialized_view<MachineRocket, ACompTransform>())
{

}

Directly?

Curious why we're forced to do the boilerplate of the view.get<>() call after we iterate the entity objects?

If it's not built into EnTT, we could probably make an iterator type fairly easily that would accomplish the task.

Reply:
I've wondered about that too, that would be pretty slick. I'll make an issue for it and we can do it later.